tizen/src/platform_tizen.cpp

```diff
@@ -225,6 +227,46 @@ std::string systemFontFallbackPath(int _importance, int _weightHint) {
     return "";
 }
 
+// No system fonts implementation (yet!)
+std::string systemFontPath(const std::string& _name, const std::string& _weight,
+                           const std::string& _face) {
+
+    std::string fontFile;
+    FcValue fcFamily, fcFace, fcWeight;
+
+    fcFamily.type = fcFace.type = fcWeight.type = FcType::FcTypeString;
+    fcFamily.u.s = (const FcChar8*)(_name.c_str());
+    fcWeight.u.s = (const FcChar8*)(_weight.c_str());
+    fcFace.u.s = (const FcChar8*)(_face.c_str());
+
+    FcConfig* config = FcInitLoadConfigAndFonts();
```

Comment:
Yeah I am taking care of this in subsequent commits which also implement fallback fonts support.